CuratedContent/TuringWay.md

```diff
@@ -0,0 +1,33 @@
+# The Turning Way: Reproducible and Ethical Science. 
+<!--deck text start-->
+An online handbook to reproducible, ethical, and collaborative computational and data science.
+<!--deck text end-->
+
+#### Contributed by [Jed Brown](https://github.com/jedbrown), [Mark C. Miller](https://github.com/markcmiller86 "Mark C. Miller GitHub Profile")
+#### Publication date: June 22, 2023
+
+Resource information | Details
+:--- | :--- 
+Article title  | [The Turing Way](https://the-turing-way.netlify.app/index.html), [DOI (most up to date)](http://doi.org/10.5281/zenodo.3332807)
+Authors | The Turing Way Community, Becky Arnold, Louise Bowler, Sarah Gibson, Patricia Herterich, Rosie Higman, … Kirstie Whitaker. (2019, March 25). The Turing Way: A Handbook for Reproducible Data Science (Version v0.0.4). Zenodo. 
+Length | Multi-Chapter, online handbook
+Focus | Reproducible, ethical and collaborative data science
+
+This is a wonderful handbook to reproducible, ethical, and collaborative computational and data science.
+It is an online collaborative project that brings together the expertise and experiences of various contributors in the field of data science and research.
+It contains impactful illustrations and is written in a consistent, approachable style.
+Although it does not have an HPC focus and some of the details are specific to biomedical applications, the chapters on Reproducible Research, Project Design, Communication, Collaboration, and Ethical Research are largely applicable to HPC/CSE.
+
+*The Turing Way* project is open source, open collaboration, and community-driven.
+It involves and supports a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
+It's goal is to provide all the information that researchers and data scientists in academia, industry and the public sector need to ensure that the projects they work on are easy to reproduce and reuse.
+
+All stakeholders, including researchers, software engineers, project leaders and funding teams, are encouraged to use *The Turing Way* to understand their roles and responsibility of reproducibility in data science.
+The moonshot goal of *The Turing Way* is to make reproducibility *too easy not to do*.
```

Comment:
Possibly `>` quote this since it's their words and written in their voice?

Reply:
Done